inception-concept-linking/src/main/resources/META-INF/asciidoc/user-guide/projects_concept-linking.adoc

```diff
@@ -0,0 +1,38 @@
+[[sect_projects_concept-linking]]
+=== Concept Linking
+The user can setup a local or remote KB for his project in the settings, which includes the
+configuration for the various IRIs. The necessary IRIs are as follows.
```

Comment:
It is unimportant that these are required, because we check that no matter what on creating a new KB.